types/node/fs.d.ts

```diff
@@ -858,7 +858,7 @@ declare module "fs" {
      * @param path A path to a file. If a URL is provided, it must use the `file:` protocol.
      * @param options If called with `withFileTypes: true` the result data will be an array of Dirent.
      */
-    function readdir(path: PathLike, options: { withFileTypes: true }, callback: (err: NodeJS.ErrnoException | null, files: Dirent[]) => void): void;
+    function readdir(path: PathLike, options: { encoding?: string | null; withFileTypes: true }, callback: (err: NodeJS.ErrnoException | null, files: Dirent[]) => void): void;
```

Comment:
According to the document，`encoding` has a default value `utf8` and withFileTypes has a default value `false`

So I think here should be `{ encoding?: string, withFileTypes?: boolean } `

Reply:
Overloaded signatures were already provided to effectively support `{ encoding?: string, withFileTypes?: boolean }`.
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e8c7805c47489ed1b20821d9c22fae10be014e75/types/node/fs.d.ts#L821-L861

Reply:
ok,you are right.and please put tests for it.

Reply:
ok,I will approve it.